awsutil/rotate.go

```diff
@@ -62,11 +62,11 @@ func (c *CredentialsConfig) RotateKeys(opt ...Option) error {
 // CreateAccessKey creates a new access/secret key pair.
 //
 // Supported options: WithEnvironmentCredentials, WithSharedCredentials,
-// WithAwsSession, WithUsername
+// WithAwsSession, WithUsername, WithTimeout
```

Comment:
This is a bit nonintuitive because it suggests that the timeout is related to creating the key itself, e.g. a timeout on that actual creation call. It's probably worth adding an extra bit of commenting around what WithTimeout does here (so anyone looking at it on pkg.dev.go will see it, whereas they likely won't see the comment below).

Reply:
Done in latest commit!